_includes/releases/v21.2/v21.2.8.md

```diff
@@ -0,0 +1,101 @@
+- When users run [`SHOW BACKUP`](../v21.2/show-backup.html) on an encrypted incremental backup, they must now set the `encyrption_info_dir` directory to the full backup directory in order for `SHOW BACKUP` to work. [#78141][#78141]
+- The [stats compaction](../v21.2/cost-based-optimizer.html#table-statistics) scheduled job no longer causes intent buildup. [#78583][#78583]
+- Implemented a scheduled logger used to capture index usage statistics to the telemetry logging channel. [#78522][#78522]
+
+<h3 id="v21-2-8-operational-changes">Operational changes</h3>
+
+- The setting `kv.bulk_io_write.min_capacity_remaining_fraction` can now be set to cause bulk ingest operations like [`IMPORT`](../v21.2/import.html), [`RESTORE`](../v21.2/restore.html), or [`CREATE INDEX`](../v21.2/create-index.html) to fail rather than write to a node that is running out of disk space. [#78575][#78575]
+- Improved jobs system resilience to scheduled jobs that may lock up jobs/scheduled job table for long periods of time. Each schedule now has a limited amount of time to complete its execution. The timeout is controlled via the `jobs.scheduler.schedule_execution.timeout` [cluster setting](../v21.2/cluster-settings.html). [#77620][#77620]
+
+
+<h3 id="v21-2-8-command-line-changes">Command-line changes</h3>
+
+- The `cockroach debug tsdump` command now allows viewing timeseries data even in cases of node failure by allowing users to rerun the command with the import filename set to `"-"`. [#77976][#77976]
+- Fixed a bug where running [`cockroach demo`](../v21.2/cockroach-demo.html) with the `--global` flag would not simulate latencies correctly when combined with the `--insecure` flag. [#78170][#78170]
+
+<h3 id="v21-2-8-db-console-changes">DB Console changes</h3>
+
+- Added a **Hot Ranges** page and linked to it in the sidebar. [#77594][#77594]
+- The `_status/nodes` endpoint is now available to all users with the `VIEWACTIVITY` role option, not just `admin` users. In the DB Console, the **Nodes Overview** and **Node Reports** pages will now display unredacted information containing node hostnames and IP addresses for all users with the `VIEWACTIVITY` role option. [#78275][#78275]
+- Fixed a bug where a node in the `UNAVAILABLE` state would not have latency defined and caused the network page to crash. [#78627][#78627]
+
+<h3 id="v21-2-8-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused errors when attempting to create table statistics (with [`CREATE STATISTICS`](../v21.2/create-statistics.html) or [`ANALYZE`](../v21.2/explain-analyze.html)) for a table containing an index that indexed only virtual [`computed columns`](../v21.2/computed-columns.html). This bug has been present since version 21.1.0. [#77565][#77565]
```

Comment:
```suggestion
- Fixed a bug that caused errors when attempting to create table statistics (with [`CREATE STATISTICS`](../v21.2/create-statistics.html) or [`ANALYZE`](../v21.2/explain-analyze.html)) for a table containing an index that indexed only virtual [`computed columns`](../v21.2/computed-columns.html). This bug has been present since v21.1.0. [#77565][#77565]
```